Src/BootCamp.Chapter/Program.cs

```diff
@@ -6,7 +6,36 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            Console.WriteLine("Hi! I will calculate your BMI. I just need to ask a few questions. What is your full name?");
+            string fullname = Console.ReadLine();
+            Console.WriteLine("How old are you?");
+            int age = Convert.ToInt32(Console.ReadLine());
+            Console.WriteLine("How much do you weigh in kg?");
+            double weight = Convert.ToDouble(Console.ReadLine());
+            Console.WriteLine("What is your height in cm?");
+            double heightcm = Convert.ToDouble(Console.ReadLine());
+            double heightm = heightcm / 100;
+            double heightsq = heightm * heightm;
```

Comment:
Unlike other variables, this seems unnecessary. Just to to put bmi into a formula here. I would recommend renaming this variable to bmi and calculating it here.